client/driver/env/env.go

```diff
@@ -124,8 +127,8 @@ func (t *TaskEnvironment) Build() *TaskEnvironment {
 	}
 
 	// Build the IP
-	if t.ip != "" {
-		t.taskEnv[TaskIP] = t.ip
+	if len(t.networks) > 0 {
+		t.taskEnv[TaskIP] = t.networks[0].IP
```

Comment:
We can probably drop this all together since we are passing in all the addresses